exercises/practice/robot-name/robot-name.spec.wren

```diff
@@ -0,0 +1,88 @@
+import "wren-testie/testie" for Testie, Expect
+import "./robot-name" for Robot
+
+class RobotNameChecker {
+  static isValid(name) {
+    return  name.count == 5  &&
+            isAlpha(name[0]) &&
+            isAlpha(name[1]) &&
+            isDigit(name[2]) &&
+            isDigit(name[3]) &&
+            isDigit(name[4])
+  }
+  static isAlpha(char) { "ABCDEFGHIJKLMNOPQRSTUVWXYZ".contains(char) }
+  static isDigit(char) { "0123456789".contains(char) }
+}
+
+
+var testSuite = Testie.new("Robot Name") { |do, skip|
+  do.test("robot has a name") {
+    var robot = Robot.new()
+    Expect.that(
+      RobotNameChecker.isValid(robot.name)
+    ).toBe(true)
+  }
+
+  skip.test("name sticks") {
+    var robot = Robot.new()
+    var name = robot.name
+    Expect.value(name).toEqual(robot.name)
+  }
+
+  skip.test("reset changes name") {
+    var robot = Robot.new()
+    var name = robot.name
+    robot.reset()
+    Expect.that(name == robot.name).toBe(false)
+  }
+
+  skip.test("reset before name called does not cause an error") {
+    var robot = Robot.new()
+    robot.reset()
+    Expect.that(
+      RobotNameChecker.isValid(robot.name)
+    ).toBe(true)
+  }
+
+  skip.test("reset multiple times") {
+    var robot = Robot.new()
+    var names = {}
+    for (i in (1..5)) {
+      robot.reset()
+      names[robot.name] = 1
+    }
+    Expect.value(names.count).toEqual(5)
+  }
+
+  skip.test("different robots have different names") {
+    var r1 = Robot.new()
+    var r2 = Robot.new()
+    Expect.that(r1.name == r2.name).toBe(false)
```

Comment:
Isn't this exactly why we added `toNotEqual`?

Reply:
It is, but it's not released yet.

Reply:
No?  Version 3 tagged Aug 28th?

Reply:
Missed it, apologies.